core/templates/domain/admin/admin-backend-api.service.ts

```diff
@@ -134,6 +134,29 @@ export class AdminBackendApiService {
       };
     });
   }
+  sendMathSvgsToBackend(latexToSvgMapping): Promise<Object> {
+    let body = new FormData();
+    for (var expId in latexToSvgMapping) {
+      for (var latexValue in latexToSvgMapping[expId]) {
+        // We need to append each file to the body with the key as the temporary
+        // latexId. In the backend, the image for each LaTeX string will be
+        // retrieved from the request body using this latexId. We cannot
+        // directly append the LaTeX strings in the request body as a key for
+        // file, because the LaTeX strings won't be always encoded properly
+        // this happens especially when the LaTeX strings has multiple back
+        // slashes. This problem of improper encoding doesn't happend if we
+        // add the LaTeX strings and its latexId in the payload and use latexId
+        // as a key for adding and retrieving raw images from the request body.
```

Comment:
This is too verbose (and a bit redundant). Can you change this to:
```suggestion
        // LaTeX strings cannot be appended in the request body as keys for
        // files because of encoding issues (eg. multiple backslash in the LaTeX
        // string is encoded as <problematic string>, <links to resources
        // supporting your claim>. As a workaround, we use a temporary latexId
        // as keys for adding and retrieving raw images from the request body.
        // Images can be extracted based on the latexId in the backend.
```

Reply:
done